1-js/06-advanced-functions/05-global-object/article.md

```diff
@@ -5,7 +5,7 @@ The global object provides variables and functions that are available anywhere.
 
 In a browser it is named `window`, for Node.js it is `global`, for other environments it may have another name.
 
-Recently, `globalThis` was added to the language, as a standardized name for a global object, that should be supported across all environments. It's supported in all major browsers.
+Recently, `globalThis` was added to the language as a standardized name for a global object, that should be supported across all environments. It's supported in all major browsers.
```

Comment:
- I'm not a native eng speaker -

I tthink it needs to fix the hanging "that" too

omiting second comma
...name for a global object that should be supported...

adding something
...name for a global object, name that should be supported...

replacing it
...name for a global object, intended to be supported...
...name for a global object; it should be supported...

Reply:
I replaced `that should` with `intended to`